src/library/tracksuggestion/lastfm/lastfmgettracksimilartask.cpp

```diff
@@ -0,0 +1,165 @@
+#include "library/tracksuggestion/lastfm/lastfmgettracksimilartask.h"
+
+#include <QDebug>
+#include <QMetaMethod>
+#include <QString>
+#include <QXmlStreamReader>
+
+#include "util/assert.h"
+#include "util/logger.h"
+#include "util/thread_affinity.h"
+
+namespace mixxx {
+
+namespace {
+
+const Logger kLogger("LastfmGetTrackSimilarTask");
+
+// API account details
+// Website:             https://www.last.fm/api
+// Application name: 	Mixxx
+// Description:         Mixxx is free and open-source software for DJing.
+// API key: 	        7830e11363f1a74c9cdb126aae9d84af
+// Shared secret: 	51671e36013c555ed91e8bb5080f50f3
+// Registered to: 	fatihemreyildiz
+// Created:             Saturday 11 Jun 2022, 11:29am
```

Comment:
How is this men to work? Do the suggestions change according to the listeners habits? Doe we finally require a personal account for each user?

Reply:
This meant to work with all the user's listening habits on last.fm. We don't need to require a personal account for any user. It doesn't require any authentication for this end point. Can be seen [on here](https://www.last.fm/api/show/track.getSimilar)

Reply:
OK, maybe you should add this info here as well.

Reply:
I will add it :+1: